flexmeasures/data/tests/conftest.py

```diff
@@ -232,3 +231,349 @@ def setup_annotations(
         asset=asset,
         sensor=sensor,
     )
+
+
+@pytest.fixture(scope="module")
+def create_solar_plants(db, setup_accounts, setup_sources) -> dict[str, Sensor]:
+    """Create Solar Plants and their Power and Price sensor along with their beliefs."""
+    asset_type = GenericAssetType(
+        name="Solar",
+    )
+    db.session.add(asset_type)
+    Solar1 = GenericAsset(
+        name="solar-1",
+        generic_asset_type=asset_type,
+    )
+    db.session.add(Solar1)
+    testing_sensor1 = Sensor(
```

Comment:
I suggest to use more self explanatory variable names. In this case, I would use `solar1_production_price_sensor` instead of `testing_sensor1`.